jerry-core/ecma/operations/ecma-regexp-object.h

```diff
@@ -29,26 +29,30 @@
  * @{
  */
 
-#define RE_EXECUTE_RECURSION_LIMIT  1000  /* Limit of RegExp executor recursion depth */
-#define RE_EXECUTE_MATCH_LIMIT      10000 /* Limit of RegExp execetur matching steps */
+/**
+ * Limit of RegExp executor recursion depth
+ */
+#define RE_EXECUTE_RECURSION_LIMIT  1000
```

Comment:
I cannot judge that it stable or not. There is an open issue for stress testing #333, but if you say that it's stable, then I believe you. :)